package.json

```diff
@@ -14,21 +14,35 @@
     "categories": [
         "Programming Languages"
     ],
+    "license": "GPL-3.0",
     "icon": "img/cylc-logo.png",
     "contributes": {
-        "languages": [{
-            "id": "cylc",
-            "aliases": ["Cylc", "cylc"],
-            "filenamePatterns": ["suite*.rc", "suite*.rc.*"],
-            "configuration": "./cylc.language-configuration.json"
-        }],
-        "grammars": [{
-            "language": "cylc",
-            "scopeName": "source.cylc",
-            "path": "./syntaxes/cylc.tmLanguage.json",
-            "embeddedLanguages": {
-                "source.jinja": "jinja"
+        "languages": [
+            {
+                "id": "cylc",
+                "aliases": [
+                    "Cylc",
+                    "cylc"
+                ],
+                "filenamePatterns": [
+                    "suite*.rc",
+                    "suite*.rc.*"
```

Comment:
Did we want to add the new filename patterns now, or wait until they have been implemented?

Reply:
Is that for Cylc8?

Reply:
Yep, suite.rc will become flow.cylc

Reply:
The config file name changes are listed here...
(https://github.com/cylc/cylc-admin/blob/16b52f057c944d1fb15d81ed6827050d6617363b/docs/proposal-config-changes.md)

Reply:
Wish I had known about that file sooner!

Reply:
I think it should be fine to add that in a later version

Reply:
Yep, shouldn't be too onerous!

Reply:
Yep but we can do that later, after the change all cylc config files will have the `.cylc` extension which will make cylc file identification a lot easier.